src/python/pants/option/arg_splitter.py

```diff
@@ -149,7 +149,7 @@ def add_scope(s):
     passthru = []
     passthru_owner = None
 
-    self._unconsumed_args = list(reversed(sys.argv if args is None else args))
+    self._unconsumed_args = list(str(a) for a in reversed(sys.argv if args is None else args))
```

Comment:
This won't work as expected in Py3. `str()` will include the byte literal.

```python
In: str('hi'.encode('utf-8'))
Out: "b'hi'"
```

Instead, we have to either use `.decode('utf-8')`, `ensure_text()`, or `if PY2: x = x.decode('utf-8')`.